larray/core/ufuncs.py

```diff
@@ -6,8 +6,41 @@
 from larray.core.array import LArray, make_args_broadcastable
 
 
-def broadcastify(func):
-    # intentionally not using functools.wraps, because it does not work for wrapping a function from another module
+def wrap_elementwise_array_func(func):
+    """
+    Wrap a function using numpy arrays to work with LArray arrays instead.
+
+    Parameters
+    ----------
+    func : function
+        A function taking numpy arrays as arguments and returning numpy arrays of the same shape. If the function
+        takes several arguments, this wrapping code assumes the result will have the combination of all axes present.
+        In numpy talk, arguments will be broadcasted to each other.
+
+    Returns
+    -------
+    function
+        A function taking LArray arguments and returning LArrays.
+
+    Examples
+    --------
+    For example, if we want to apply the Hodrick-Prescott filter from statsmodels we can use this:
+
+    >>> from statsmodels.tsa.filters.hp_filter import hpfilter         # doctest: +SKIP
+    >>> hpfilter = wrap_elementwise_array_func(hpfilter)               # doctest: +SKIP
+
+    hpfilter is now a function taking a one dimensional LArray as input and returning a one dimensional LArray as output
+
+    Now let us suppose we have a ND array such as:
+
+    >>> from larray.random import normal
+    >>> arr = normal(axes="sex=M,F;year=2010..2020")            # doctest: +SKIP
+
+    We can apply an Hodrick-Prescott filter to it by using:
+
+    >>> # 6.25 is the recommended smoothing value for annual data
+    >>> cycle, trend = arr.apply(hpfilter, 6.25, axes="year")             # doctest: +SKIP
+    """
```

Comment:
This example should maybe be repeated in the Tutorial ?

Reply:
yes, good idea, but I do not think it is worth delaying the release for it.

Reply:
The tutorial needs a large update anyway. That indeed can be done after the coming release.